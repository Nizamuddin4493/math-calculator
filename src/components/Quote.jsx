import React from 'react';

function Quote() {
  return (
    <>
      <section className="mian-txt-body">
        <h1>Quote</h1>
        <br />
        <article>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis fuga
          consequuntur cum! Fugit quia cupiditate necessitatibus neque
          blanditiis placeat accusamus sed aspernatur facilis minima ullam incidunt.
          Hic molestias quae odio.
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis fuga
          consequuntur cum! Fugit quia cupiditate necessitatibus neque
          blanditiis placeat accusamus sed aspernatur facilis minima ullam incidunt.
          Hic molestias quae odio.
        </article>
      </section>
    </>
  );
}

export default Quote;
