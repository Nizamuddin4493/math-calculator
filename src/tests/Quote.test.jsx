import { BrowserRouter as Router } from 'react-router-dom';
import TestRenderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders correctly', () => {
  const tree = TestRenderer.create(<Router><Quote /></Router>).toJSON();
  expect(tree).toMatchSnapshot();
});
