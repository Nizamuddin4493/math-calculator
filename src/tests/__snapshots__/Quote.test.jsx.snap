// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="mian-txt-body"
>
  <h1>
    Quote
  </h1>
  <br />
  <article>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis fuga consequuntur cum! Fugit quia cupiditate necessitatibus neque blanditiis placeat accusamus sed aspernatur facilis minima ullam incidunt. Hic molestias quae odio. Lorem ipsum dolor sit amet consectetur adipisicing elit. Blanditiis fuga consequuntur cum! Fugit quia cupiditate necessitatibus neque blanditiis placeat accusamus sed aspernatur facilis minima ullam incidunt. Hic molestias quae odio.
  </article>
</section>
`;
